CONTRIBUTING.md

```diff
@@ -46,16 +46,16 @@ Should you choose to work on a new issue or an existing issue from <https://gith
 
 - Make the required changes to the component.
 - Write integrations tests and unit tests.
-- Check for all lint errors, if there are any fix it.
+- Check for all lint errors, if there are any fix them.
 - Run tests for all the components.
 - If the component you modify is used in another component make sure you update the tests for that component as well.
-- Raise a Pull Requesst to the Crayons Repository.
+- Raise a Pull Request to the Crayons Repository.
 
 ### Commit the changes to Github
 
-Crayons repository uses github pre commit hooks, follow the below steps to commit our changes to github
+Crayons repository uses github pre commit hooks, follow the below steps to commit your changes to github
 
-1. Make your changes to the file, add the untracked files to staging area.
+1. Make your changes to the file and add the untracked files to staging area.
```

Comment:
Could we modify this to one of the following?

- Modify the requisite files and add the untracked files to the staging area.

- Make changes to the files and add the untracked files to the staging area.